frontend/app/pages/submit_personal_info.vue

```diff
@@ -0,0 +1,199 @@
+<script setup lang="ts">
+import type { FormSubmitEvent } from '#ui/types'
+import { FetchError } from 'ofetch'
+import * as z from 'zod'
+
+definePageMeta({
+  layout: 'centered',
+})
+
+type Schema = z.output<typeof schema>
+
+const current_year = new Date().getFullYear()
+
+enum Age {
+  '10-13' = 10,
+  '14-16' = 14,
+  '16-18' = 16,
+  '19-22' = 19,
+  '23+' = 23,
+}
+
+function matchAge(age: Age) {
+  return current_year - age
+}
+
+const ageValuesReversed = Object.keys(Age).filter(key => !Number.isNaN(Number(key))).reverse()
+
+function getAge(birth_year: number) {
+  const age = current_year - birth_year
+
+  for (const value of ageValuesReversed) {
+    if (age >= Number(value)) {
+      return String(Age[value as keyof typeof Age])
+    }
+  }
+
+  return undefined
+}
+
+const ageKeys = Object.keys(Age).filter(key => Number.isNaN(Number(key)))
```

Comment:
The order of stuff here is completly random, either remove this const or move it just abvoe